### PR TITLE
Customer on projects.update [PRO-4644]

### PR DESCRIPTION
**Pull request by adognameddog**

### CODE REVIEW

File: src/07-projects/projects.apib [hunk 1]
```diff
@@ -205,6 +205,12 @@ Update a project.
                 + done
                 + cancelled
         + starts_on: `2016-02-04` (string)
+        + customer (object) - Null will unlink the customer from the project
```
Comment (mrtus) [inline src/07-projects/projects.apib]: Q: Reading the extra information given, did you mean to make customer nullable?
```apib
        + customer (object, nullable) - Null will unlink the customer from the project
```

Comment (adognameddog) [reply src/07-projects/projects.apib]: I did that, but then I saw this does nothing in the generated documentation, it does not mention it is nullable 🤷

**Review by joaoinacio — APPROVED** [review]

**Comment by joaoinacio** [comment]
✅ Manually checked.

File: src/07-projects/projects.apib (hunk 1 above)
Comment (mrtus) [reply src/07-projects/projects.apib]: That is weird indeed 🤔 I thought this was visible somewhere, it should be at least in the Json Schema itself.

**Review by mikeverf — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**